### PR TITLE
Added password reset browser test + fixes

### PR DESCRIPTION
**Pull request by AlexVanderbist**
For #335 

- Added missing email to url in `setPassword` mail
- Fixed action in password reset form
- Added test for back user password reset flow
- Added `.env.backup` to `.gitignore`

### CODE REVIEW

File: tests/Browser/Back/PasswordResetTest.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                ->assertSee('Wachtwoord vergeten?')
+                ->clickLink('Wachtwoord vergeten?')
+                ->assertPathIs('/blender/password/reset');
+        });
+    }
+
+    /** @test */
+    public function it_can_reset_passwords()
+    {
+        $user = User::create(['email' => 'test@example.com', 'first_name' => 'Test', 'last_name' => 'Test', 'status' => 'active']);
```
Comment (sebastiandedeyne) [inline tests/Browser/Back/PasswordResetTest.php]: This line's a bit long, add a newline per attribute.

File: tests/Browser/Back/PasswordResetTest.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+            $browser
+                ->visit($passwordResetUrl)
+                ->assertSee('WACHTWOORD WIJZIGEN')
+                ->type('password', 'secret')
+                ->type('password_confirmation', 'secret')
+                ->press('Wachtwoord instellen')
+                ->assertPathIs('/blender')
+                ->assertSee(__('passwords.reset'));
+        });
```
Comment (sebastiandedeyne) [inline tests/Browser/Back/PasswordResetTest.php]: Could you add a database assertion afterwards, to check if the password has actually changed?

**Review by sebastiandedeyne — CHANGES_REQUESTED** [review]

File: tests/Browser/Back/PasswordResetTest.php [hunk 3]
```diff
@@ -40,11 +48,15 @@ public function it_can_reset_passwords()
...
+                ->type('password', $newPassword)
+                ->type('password_confirmation', $newPassword)
                 ->press('Wachtwoord instellen')
                 ->assertPathIs('/blender')
                 ->assertSee(__('passwords.reset'));
         });
+
+        $user = User::where('email', $user->email)->first()->makeVisible('password');
+
+        $this->assertTrue(Hash::check($newPassword, $user->password));
```
Comment (sebastiandedeyne) [inline tests/Browser/Back/PasswordResetTest.php]: I don't think `makeVisible` is necessary here. Line 58 could be omitted, and the password can be fetched using `$user->fresh()->password` (I think)

Comment (AlexVanderbist) [reply tests/Browser/Back/PasswordResetTest.php]: Works perfectly fine, thanks!

**Comment by sebastiandedeyne** [comment]
👌

**Pull request merged**